### PR TITLE
central entropy output added

### PR DESCRIPTION
**Pull request by mari2895**
## PR Summary
added output for entropy at z=0

## PR Checklist

- [ ] Adds a test for any bugs fixed. Adds tests for new features.
- [x] Format your changes by calling `scripts/bash/format.sh`.
- [x] Explain what you did.

### CODE REVIEW

File: src/phoebus_driver.cpp [hunk 1]
```diff
@@ -1068,6 +1069,10 @@ void UserWorkBeforeOutput(MeshBlock *pmb, ParameterInput *pin) {
         Real gam1[3][3];
         Real gam2[3][3];
         Real gam3[3][3];
+        const Real z = coords.Xc<3>(k, j, i);
+        const Real sigma = 2 * coords.CellWidthFA(X3DIR, k, j, i);
```
Comment (Yurlungur) [inline src/phoebus_driver.cpp:1073]: this depends on the mesh width on the block you're on which makes me a bit uncomfortable. I suggest making sigma a free parameter that is set-able in the input deck.

File: src/phoebus_driver.cpp [hunk 2]
```diff
@@ -1068,6 +1069,10 @@ void UserWorkBeforeOutput(MeshBlock *pmb, ParameterInput *pin) {
...
+        const Real s0 = s * std::exp(-z * z / sigma / sigma);
```
Comment (Yurlungur) [inline src/phoebus_driver.cpp:1074]: Alternatively (or possibly additionally) we should probably normalize our Gaussian

**Review by Yurlungur — CHANGES_REQUESTED** [review]

File: src/phoebus_driver.cpp (hunk 1 above)
Comment (mari2895) [reply src/phoebus_driver.cpp:1073]: hmm that's a weird thing to have in input deck... plus out of context from everything else, no?

File: src/phoebus_driver.cpp (hunk 2 above)
Comment (mari2895) [reply src/phoebus_driver.cpp]: done

File: src/phoebus_driver.cpp (hunk 1 above)
Comment (mari2895) [reply src/phoebus_driver.cpp]: done

File: src/fluid/fluid.cpp [hunk 3]
```diff
@@ -207,6 +207,7 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   physics->AddField(p::entropy::name(), mprim_scalar);
   physics->AddField(p::cs::name(), mprim_scalar);
   physics->AddField(diag::ratio_divv_cs::name(), mprim_scalar);
+  physics->AddField(diag::entropy_z_0::name(), mprim_scalar);
```
Comment (Yurlungur) [inline src/fluid/fluid.cpp:210]: might make more sense to register this var in the analysis initialize function. But this is fine for now.

File: src/analysis/analysis.cpp [hunk 4]
```diff
@@ -0,0 +1,13 @@
+#include "phoebus_utils/variables.hpp"
+#include "analysis.hpp"
+
+namespace analysis{
+std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
+  auto analysis_pkg = std::make_shared<StateDescriptor>("analysis");
+  Params &params = analysis_pkg->AllParams();
+  Real sigma = pin->GetOrAddReal("analysis", "sigma", 2e-3);
+  params.Add("sigma", sigma);
```
Comment (Yurlungur) [inline src/analysis/analysis.cpp:9]: Maybe add a check that sigma > 0

**Review by Yurlungur — APPROVED** [review]

File: src/analysis/analysis.cpp (hunk 4 above)
Comment (mari2895) [reply src/analysis/analysis.cpp:12]: done

**Pull request merged**